src/components/dev-hub/podcast-player/player-listen-on.js

```diff
@@ -0,0 +1,61 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+
+import AppleLogo from '~components/dev-hub/icons/audio-player/logos/apple-logo';
+import Button from '~components/dev-hub/button';
+import GoogleLogo from '~components/dev-hub/icons/audio-player/logos/google-logo';
+import SpotifyLogo from '~components/dev-hub/icons/audio-player/logos/spotify-logo';
+
+import { colorMap, size } from '~components/dev-hub/theme';
+
+export const LogosContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    justify-content: center;
+`;
+
+const StyledButton = styled(Button)`
+    margin-right: 30px;
+    padding: ${size.tiny};
+
+    &:hover {
+        svg > path {
+            fill: ${colorMap.greyLightOne};
+        }
+    }
+
+    &:active {
+        svg > path {
+            fill: #707f81;
+        }
+    }
+`;
+
+const PlayerListenOn = ({
+    appleClick,
+    className,
+    googleClick,
+    spotifyClick,
+}) => (
+    <LogosContainer className={className}>
+        <StyledButton aria-label="google-podcasts" onClick={googleClick}>
```

Comment:
I have the link for this, it can go to `https://podcasts.google.com/feed/aHR0cHM6Ly9tb25nb2RiLmxpYnN5bi5jb20vcnNz?sa=X&ved=2ahUKEwiE97rnkfDrAhU1n3IEHanUCRUQ9sEGegQIARAC`